# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/icc/pkg_gawk_memcpy.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./missing_d/memcpy.c.orig	Fri Jun 21 18:50:43 2002
+++ ./missing_d/memcpy.c	Fri Jun 21 18:51:02 2002
@@ -4,6 +4,7 @@
  * We supply this routine for those systems that aren't standard yet.
  */
 
+/*
 char *
 memcpy (dest, src, l)
 register char *dest, *src;
@@ -16,3 +17,4 @@
 
 	return ret;
 }
+*/
